src/main/webapp/form.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Student Recommendations</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/styles/fonts.css">
+    <link rel="stylesheet" href="/styles/common.css">
+    <link rel="stylesheet" href="/styles/form.css">
+  </head>
+  <body>
+    <div class="content">
+      <div class="container">
+        <h1>StudentRecs</h1>
+        <h3>a recommendation system for students, by students!</h3>
+      </div>
+    </div>
+    <form name="user-details" action="/user" method="POST">
+      <p>Hello! my name is</p>
+      <input type="text" name="name" class="name">
+      <br><br>
+      <p>I am currently a student in department of</p>
```

Comment:
nit: this should be "I am currently a student in the department of"

Reply:
thank you for the correction!